user.lua

```diff
@@ -62,7 +62,6 @@ function User:update(dt)
     local m_x, m_y = love.mouse.getPosition()
 
     if dir and dir ~= self.cdir then
-        self.cdir = cdir
```

Comment:
I don't know what was going on here; `cdir` is never set anywhere else, so this was effectively setting `self.cdir` to nil. I thought it might be a typo and tried `self.cdir = dir` but that broke everything. I'm like 75% sure that removing it is the right thing but worth a double-check; it's possible it should be `self.cdir = nil` if that matters.

Reply:
Fixed in 6dccc09.